### PR TITLE
Add documentation on jQuery chaining

### PR DESCRIPTION
**Pull request by Orrison**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

Add documentation on how to chain jQuery methods.

### CODE REVIEW

File: guide/english/jquery/jquery-chaining/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+$('#someElement').removeClass('classA');
+$('#someElement').addClass('classB');
+```
+Into a single statement:
+```javascript
+$('#someElement').removeClass('classA').addClass('classB');
+```
+
+#### More Information:
+For more information, please visit the <a href='https://www.w3schools.com/jquery/jquery_chaining.asp' target='_blank' rel='nofollow'>w3schools documentation</a>.
```
Comment (thecodingaviator) [inline guide/english/jquery/jquery-chaining/index.md]: Please change this line to the markdown link format

Comment (Orrison) [reply guide/english/jquery/jquery-chaining/index.md]: Fixed in my recent commit

**Comment by RandellDawson** [comment]
@thecodingaviator new changes good fine.

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thanks @Orrison  for contributing to this repo! 👍

We're glad to accept your changes and look forward to more contributions from you!

**Pull request merged**